components/menu/index.en-US.md

```diff
@@ -0,0 +1,69 @@
+---
+category: Components
+chinese: 导航菜单
+cols: 1
+type: Navigation
+english: Menu
+---
+
+Menu list of Navigation.
+
+## When to use
+
+Navigation menu is important for a website, which makes users jumping from one page to another. Mostly it includes top navigation and side navigation. Top navigation provides all the category and functions of the website. Side navigation provides the Multi-level structure of the website.
+
+More layout and samples: [layout](/docs/spec/layout).
+
+```html
+<Menu>
+  <Menu.Item>Menu</Menu.Item>
+  <SubMenu title="SubMenu">
+    <Menu.Item>SubMenuItem</Menu.Item>
+  </SubMenu>
+</Menu>
+```
+
+## API
+
+### Menu props
+
+| Param    | Description   | Type     | Default value       |
+|----------|---------------|----------|--------------|
+| theme    | color of the theme | String: `light` `dark` | `light` |
+| mode | type of the menu, support vertical, horizontal and inline mode | String: `vertical` `horizontal` `inline` | `vertical` |
```

Comment:
| mode | type of the menu; vertical, horizontal, and inline modes are supported | String: `vertical` `horizontal` `inline` | `vertical` |